src/App.vue

```diff
@@ -456,6 +456,7 @@ export default {
       } else {
         this.$refs.builder.refreshContent();
       }
+      this.$store.commit("globalErrorsModule/setMode", this.mode);
```

Comment:
To improve this code you could try to map the `setMode` mutation
see: https://blog.logrocket.com/best-practices-mapping-vuex/#vuexmapmutations
something like:
```
import { mapMutations } from 'vuex'

export default {
  methods: {
    ...mapMutations("globalErrorsModule", [
      "setMode", // map `this.setMode()` to `this.$store.commit('globalErrorsModule/setMode')`
    ]),
  }
}
```